plugins/notifiers/client.go

```diff
@@ -19,7 +20,8 @@ type Client interface {
 }
 
 const (
-	ProviderTypeSlack = "slack"
+	ProviderTypeSlack       = "slack"
+	DefaultTimeoutInSeconds = 10
```

Comment:
this also can be part of `pkg/http`

Reply:
Timeout is the part of the new HTTP client from Go package. Let's just remove the const and use the default from the http package (90 secs). But still this 10 secs should be set on config since 90 is too much for the timeout.